packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from "@typescript-eslint/experimental-utils";
+
+export function isLoadFunction(node: TSESTree.MemberExpression): boolean {
+  return (
+    node.property.type === TSESTree.AST_NODE_TYPES.Identifier &&
+    node.property.name === "load"
+  );
+}
+
+export function isLoadReference(node: TSESTree.Identifier) {
+  return (
+    node.parent &&
+    node.parent.type === TSESTree.AST_NODE_TYPES.MemberExpression &&
+    isLoadFunction(node.parent)
+  );
+}
+
+function composeObjectExpressionPropertyIntoString(
+  objectExpression: TSESTree.ObjectExpression
+): string {
+  let composedProperty: string = "";
+  objectExpression.properties.forEach((property) => {
+    if (property.type === AST_NODE_TYPES.Property) {
+      if (property.key.type === AST_NODE_TYPES.Identifier) {
+        composedProperty += property.key.name;
+      }
+      if (property.value.type === AST_NODE_TYPES.ObjectExpression) {
+        composedProperty +=
+          "/" + composeObjectExpressionPropertyIntoString(property.value);
+      }
+    }
+  });
+
+  return composedProperty;
+}
+
+export function getLoadArgument(node: TSESTree.Node | undefined): string {
```

Comment:
This method still assumes load by name and one of the argument parsing, but doesn't check that it's actually a load statement.

It also assumes a single argument, but load can have more then one.

Reply:
Yeah, I didn't know `load` could have multiple arguments until very recently.
I guess we could try to change it to accept multiple arguments in another PR because some other rules also use this function and we would need to see how they do interact with it.

Reply:
The multiple args could be done in another PR, but name should reflect what it's doing . . . extracting args, not specifically load args.  The caller is making sure it's load args.

Reply:
Now the rule is checking for load

Reply:
The rule yes . . . but not the method.  This is a utility method and so there is not context of it being part of a load call.  Another rule could call this without checking for load on another method and the name wouldn't make sense.

Reply:
Sorry, missed the check-in.  Makes sense now.